source/community/activities/summer-of-code.html.md

```diff
@@ -42,6 +42,24 @@ When students approach you about the idea you listed:
 *   If you are redirecting students from your idea, please add [No longer taking applicants] to its title in the list below.
 *   Don't hesitate to reach out to the oVirt GSoC admins if you need help redirecting students.
 
+## oVirt Ideas for Google Summer of Code 2018
+
+### **Idea** KubeVirt/qemu-guest-agent
```

Comment:
That should be under KubeVirt, not oVirt?

Reply:
is kubevirt applying independently? Or you just meant the title?

Reply:
I don't know it does - it should - please find out promptly. If not, we'll host it here for a while...

Reply:
I was under the impression it does not. @fabiand @rmohr is KubeVirt applying for google summer of code this year?